### ISSUE TITLE
Test and gracefully handle non-floating point network and module inputs

### ISSUE THREAD

**Issue by spfrommer** [spfrommer/torchexplorer#10]
Tests should include the pytorch `nn.Embedding` module. The input gradients should just not be displayed in this case.
